BUILD

```diff
@@ -7,66 +7,77 @@ load("//rules/go:index.bzl", "go_sdk_tool")
-                        "/enterprise/server/": "Enterprise server code",
-                    },
-                    "analyzer_flags": {
-                        "default-signifies-exhaustive": "true",
-                    },
-                },
-                "fieldalignment": {
-                    "only_files": {
-                        "/server/": "Server code",
-                        "/enterprise/server/": "Enterprise server code",
-                    },
-                    "exclude_files": {
-                        ".*": "Disable entirely for now",
-                        ".*\\.pb\\.go": "Auto-generated proto files",
-                    },
-                },
-                "nilness": {
-                    "only_files": {
-                        "/server/": "Server code",
-                        "/enterprise/server/": "Enterprise server code",
-                    },
-                    "exclude_files": {
-                        ".*_cgo_gotypes\\.go": "cgo internal files",
-                    },
-                },
-            } | {
                 analyzer: {
-                    "only_files": {
-                        "/server/": "Server code",
-                        "/enterprise/server/": "Enterprise server code",
-                    },
+                    config: {
+                        # Support Windows and Unix path separators
+                        path.replace("/", "[\\\\, \\/]"): reason
+                        for path, reason in paths.items()
+                    }
+                    for config, paths in configs.items()
                 }
-                for analyzer in [
-                    "assign",
-                    "cgocall",
-                    "composites",
-                    "copylocks",
-                    "lostcancel",
-                    "stdmethods",
-                    "stringintconv",
-                    "structtag",
-                    "unreachable",
-                    "unsafeptr",
-                ]
-            } | {
-                analyzer: {
-                    "exclude_files": {
-                        "external[\\\\,\\/]": "third_party",
+                for analyzer, configs in ({
+                    "exhaustive": {
+                        "exclude_files": {
+                            "external/": "third_party",
+                        },
+                        "analyzer_flags": {
+                            "default-signifies-exhaustive": "true",
+                        },
                     },
-                }
-                for analyzer in ANALYZERS
+                    "copylocks": {
+                        "exclude_files": {
+                            "external/": "third_party",
+                            # TODO(sluongng): this should be fixed on rules_go side
+                            # https://github.com/bazelbuild/rules_go/issues/3619
+                            "cgo/github.com/shirou/gopsutil/": "third_party cgo",
```

Comment:
This is getting kind of complicated. Could you add this exclude to all analyzers and put `copylocks` in the list below, and remove all the loops and stuff above here in favor of a static definition which would be easier to understand?

Reply:
Oh this is a brilliant idea!